Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use-Case-2_Two-Qubit-Randomized-Benchmarking/configuration.py

```diff
@@ -0,0 +1,552 @@
+from pathlib import Path
```

Comment:
Since the qubit indices are 0 and 1 I think it would be good to change the variable indices to 0 and 1 as well instead of 1 and 2